utils/renderTestFiles.sh

```diff
@@ -0,0 +1,102 @@
+pod_cmd="kubectl -n test"
+
+$webhoook_cmd cat /etc/webhook/config/tsiMutateConfig.yaml > ${TESTS}/ConfigFile.yaml
+$webhoook_cmd cat /tmp/ExpectTsiMutateConfig.json > ${TESTS}/ExpectTsiMutateConfig.json
+# kk exec -it $(kk get po | grep mutate-webhook| awk '{print $1}') -- cat /etc/webhook/config/tsiMutateConfig.yaml > ${TESTS}/ConfigFile.yaml
+# Add to  ExpectTsiMutateConfig.json missing annotations as:
+# "admission.trusted.identity/tsi-cluster-name": <span style="background-color: #fcff7f">"testCluster" => "minikube"</span>,
+# "admission.trusted.identity/tsi-images": <span style="background-color: #fcff7f">"" => "ubuntu@sha256:250cc6f3f3ffc5cdaa9d8f4946ac79821aafb4d3afc93928f0de9336eba21aa4"</span>,
+# "admission.trusted.identity/tsi-region": <span style="background-color: #fcff7f">"testRegion" => "eu-de"</span>
+
+$pod_cmd create -f ${EXAMPLES}/myubuntu.yaml
+# if error delete and recreate
+sleep $SLEEP
+$webhoook_cmd cat /tmp/FakeAdmissionReview.json > ${TESTS}/FakeAdmissionReview.json
+$webhoook_cmd cat /tmp/ExpectMutateInit.json > ${TESTS}/ExpectMutateInit.json
+$webhoook_cmd cat /tmp/FakeIsSafeX.json > ${TESTS}/FakeIsSafeCreateOK.json
+$webhoook_cmd cat /tmp/FakePod.json > ${TESTS}/FakePod.json
+$webhoook_cmd cat /tmp/FakeMutationRequired.json > ${TESTS}/FakeMutationRequired.json
+$webhoook_cmd cat /tmp/FakeUpdateAnnotationTarget.json > ${TESTS}/FakeUpdateAnnotationTarget.json
+$webhoook_cmd cat /tmp/FakeUpdateAnnotation.json > ${TESTS}/FakeUpdateAnnotation.json
+$webhoook_cmd cat /tmp/ExpectUpdateAnnotation.json > ${TESTS}/ExpectUpdateAnnotation.json
+sed 's/}/,/g' ${TESTS}/FakeUpdateAnnotation.json > ${TESTS}/FakeUpdateAnnotationError.json
+cat <<EOT >> ${TESTS}/FakeUpdateAnnotationError.json
+    "tsi.secrets": "- tsi.secret/name: \"mysecret1\"\n  tsi.secret/role: \"tsi-role-rcni\"\n  tsi.secret/vault-path: \"secret/tsi-rcni\"\n  tsi.secret/local-path: \"mysecrets/myubuntu-mysecret1\"\n- tsi.secret/name: \"mysecret2.json\"\n  tsi.secret/role: \"tsi-role-rcni\"\n  tsi.secret/vault-path: \"secret/tsi-r\"\n  tsi.secret/local-path: \"mysecrets/myubuntu-mysecret2\"\n- tsi.secret/name: \"password\"\n  tsi.secret/role: \"tsi-role-rcni\"\n  tsi.secret/vault-path: \"secret/tsi-r\"\n  tsi.secret/local-path: \"mysecrets/myubuntu-passwords\"\n- tsi.secret/name: \"invalid\"\n  tsi.secret/role: \"tsi-role-rcni\"\n  tsi.secret/vault-path: \"secret/tsi-rcni\"\n  tsi.secret/local-path: \"mysecrets/myubuntu-invalid\"\n- tsi.secret/name: \"non-existing\"\n  tsi.secret/role: \"tsi-role-rcni\"\n  tsi.secret/vault-path: \"secret/nothing\"\n  tsi.secret/local-path: \"mysecrets/non-existing\"\n"
+}
+EOT
+sed 's/ubuntu@sha256:250cc6f3f3ffc5cdaa9d8f4946ac79821aafb4d3afc93928f0de9336eba21aa4/ubuntu@sha256:250cc6f3f3ffc5cdaa9d8f4946ac79821aafb4d3afc93928f0de9336eba21aa4,trustedseriviceidentity\/ti-jwt-sidecar:v1.5/g' ${TESTS}/FakeUpdateAnnotation.json > ${TESTS}/FakeUpdateAnnotation2.json
```

Comment:
Does the versioning here i.e. `v1.5` have to change per iteration to work? do we need to use an env variable instead?

Reply:
I removed that and made it completely dynamic